src/anndata/_io/specs/lazy_methods.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from contextlib import contextmanager
+from pathlib import Path
+
+import h5py
+import numpy as np
+from scipy import sparse
+
+import anndata as ad
+from anndata.compat import H5Array, H5Group, ZarrArray, ZarrGroup
+
+from .registry import _LAZY_REGISTRY, IOSpec
+
+# TODO: settings
+stride = 100
+h5_chunks = 1000
```

Comment:
Not sure what to do about this.  The code path is quite complicated for i/o since we're relying on the registry.  The `settings` object would be one way to go about this.  And then another would be updating the registry to allow for passing in `kwargs`...

Reply:
I think it would make sense for there to be a `chunks` argument to read elem as Dask. Something like: `chunks: None | int | tuple[int, ...] = None`.

Reply:
I see what's up - I didn't know kwargs got passed through.  Done now.  The chunks here is much less complicated and is really `chunk_size` so should be an `int` and I also made `stride` a kwarg